src/api.js

```diff
@@ -1,7 +1,26 @@
 import axios from "axios";
 
-const fetchArticles = () => {
-  return axios.get("https://nc-news-app-bch2.onrender.com/api/articles");
+const fetchArticles = ({sort, topic, order}) => {
+  let path = `https://nc-news-app-bch2.onrender.com/api/articles`
+  if(topic || sort || order){
+    path += `?`
+  }
+  
+  if (topic){
+    path+= `topic=${topic}`
+  }
+  if(sort && topic){
+    path += `&sort_by=${sort}`
+    
+  }
+  if (sort && !topic){
+    path+= `sort_by=${sort}`
+  }
+  if(order){
+    path+=`&order=${order}`
+  }
```

Comment:
This all works well, but we can make the code a little bit neater with axios params. Take a look and see if you can refactor 👍